.github/workflows/MilestoneIssue.yml

```diff
@@ -16,7 +18,30 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const TODO_COLUMN = 4971951;
-            console.log("Creating Issue card");
-            // Action will fail if a card already exists for this issue
-            github.projects.createCard({ column_id: TODO_COLUMN, content_id: context.payload.issue.id, content_type: "Issue" });
+            const IN_PROGRESS_COLUMN = 4971952;
+            const REVIEW_IN_PROGRESS_COLUMN = 4971953;
+            const REVIEW_APPROVED_COLUMN = 4971954;
+            const DONE_COLUMN = 4971955;
+            //
+            async function findCard(content_url) {
```

Comment:
This method is copy/pasted from here, except different column ordering:
https://github.com/SonarSource/sonar-dotnet/blob/9bf155c48565506572195db880001524699662f4/.github/workflows/RequestReview.yml#L39-L53